spree_slider.gemspec

```diff
@@ -1,33 +1,42 @@
-# encoding: UTF-8
 Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
   s.name        = 'spree_slider'
-  s.version     = '3.3.0'
+  s.version     = '4.0.0'
   s.summary     = 'Spree Slider extension'
   s.description = 'Adds a slider to the homepage'
-  s.required_ruby_version = '>= 1.8.7'
+  s.required_ruby_version = '>= 2.5.0'
 
   s.author            = 'Giuseppe Privitera'
   s.email             = 'priviterag@gmail.com'
-  s.homepage          = 'https://github.com/priviterag/spree_slider'
+  s.homepage          = 'https://github.com/spree-contrib/spree_slider'
 
   s.files         = `git ls-files`.split("\n")
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  spree_version = '>= 3.1.0', '< 4.0'
-  s.add_dependency 'spree_core', spree_version
+  spree_version = '>= 4.0', '< 5.0'
+  s.add_dependency 'active_storage_validations', '~> 0.8.4'
```

Comment:
Latest version of spree relies on `active_storage_validations 0.9`

Reply:
fixed in https://github.com/spree-contrib/spree_slider/pull/50/files